finagle-http/src/main/scala/com/twitter/finagle/Http.scala

```diff
@@ -404,6 +411,12 @@ object Http extends Client[Request, Response] with HttpRichClient
     def withMaxInitialLineSize(size: StorageUnit): Server =
       configured(param.MaxInitialLineSize(size))
 
+    /**
+     * Enable the collection of HTTP specific metrics. See [[http.filter.StatsFilter]].
+     */
+    def enableHttpStats(): Server =
+      withStack(http.filter.StatsFilter.module +: stack)
```

Comment:
I don't think we want to have non-idempotent configuration methods on StackClients.  How about instead inserting a nop "http stats" module into the Http stack and then replacing it here?

It might also be useful to look at our design principles for configuration methods for StackClients.

http://twitter.github.io/finagle/guide/Configuration.html#design-principles